app/views/content/life-as-a-teacher/change-careers/making-a-career-change-to-teaching.md

```diff
@@ -14,5 +14,5 @@ keywords:
 content: 
   - "content/life-as-a-teacher/change-careers/making-a-career-change-to-teaching/header" 
   - "content/life-as-a-teacher/change-careers/making-a-career-change-to-teaching/article"
-  - "content/life-as-a-teacher/how-to-become-a-teacher-cta"
```

Comment:
It looks like we've removed all instances of this partial throughout the site now (at least I can't find any!), if that's the case, we should remove the partial itself as well unless we need to keep it for anything

Reply:
Good spot - I'll raise a separate ticket to consider this and the home page version (which is not being used either)